lib/puppet/provider/credential/wincred.rb

```diff
@@ -39,11 +39,11 @@ def username=(value)
   end
 
   def value
-    current_value[:value].force_encoding('utf-8') || :absent
+    current_value[:value] || :absent
```

Comment:
I think it should `force_encoding('utf-16le')` here to be consistent (`Identity#from_wincred` can then only reencode to utf8).